src/components/Loader/Loader.tsx

```diff
@@ -40,182 +106,32 @@ export const Loader: FC<Props & ElementProps> = ({
   className = '',
```

Comment:
この初期化いらなくなるはずなので消したい。

```suggestion
  className,
```

Reply:
https://github.com/kufu/smarthr-ui/pull/3985/commits/ccdd64db4d41aea8da6ffcda18e828515a9d8a02 😎